### ISSUE TITLE
Localization and pose estimation of objects 

### ISSUE THREAD

**Issue by mabdulmuhsin** [fizyr/keras-retinanet#891]
I have a data set containing arrows with posture and positions ,
annotation is like this: "image_path , x1 (center), y1 (center),D(degree),Arrow (class name)"
Can i use Csv Generator object to train my model using retina net ?
"I want to find a center (x,y) and degree(angle) of Arrows by using transfer learning ".
could I fit and train Retina net in this specific matter ?

**Comment by martinzlocha** [comment]
It should be possible however it will require a lot of modification to how RetinaNet works. I feel like the changes are so big that you are better-off just starting from scratch and just applying the principles from RetinaNet. Or a different approach might be to reformulate your task to instead get you the bounding boxes and then use a secondary stage to get you the degree.

**Comment by hgaiser** [comment]
@martinzlocha answered it perfectly. It is possible to do this using keras-retinanet, but you'll need your own generator to generate the correct batches and you'll need to generate some custom submodels and loss functions. Alltogether, it's possible, but requires some significant knowledge of how retinanet (and keras-retinanet) works.

Closing this as it is not an issue with our code.
